packages/fast-components-class-name-contracts-msft/src/index.ts

```diff
@@ -24,3 +24,4 @@ export * from "./slider-label";
 export * from "./subheading";
 export * from "./text-action";
 export * from "./text-field";
+export * from "./tree-view-item";

```

Comment:
no tree view class name contract?

Reply:
We typically don't dupe all contracts here - there's not a MSFT specific tree view so it simply lives in base.